src/page/rules/rules.page.html

```diff
@@ -121,7 +121,7 @@ <h3 class="mat-primary">The Accolades</h3>
 <table>
     <tr>
         <th>Participation</th>
-        <td>Anyone may enter regardless or age or experience level.<br />Editors are encouraged to attend Anime Weekend
+        <td>Anyone may enter regardless oF age or experience level.<br />Editors are encouraged to attend Anime Weekend
```

Comment:
capital F is a new typo
```suggestion
        <td>Anyone may enter regardless of age or experience level.<br />Editors are encouraged to attend Anime Weekend
```